tests/benchmark_search.js

```
@@ -21,12 +22,40 @@ class HTMLElement {
+    this.children = []; // Clear children when setting innerHTML
+  }
+
+  get textContent() { return this._textContent; }
+  set textContent(val) { this._textContent = val; }
 
   appendChild(child) {
-    this.children.push(child);
-    child.parentNode = this;
+    if (child.tagName === 'DOCUMENT_FRAGMENT') {
+      this.children.push(...child.children);
+      child.children.forEach(c => c.parentNode = this);
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)